model/value_histogram_test.go

```diff
@@ -743,3 +746,17 @@ func TestMatrixHistogramJSON(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkJSONMarshallingSampleHistogramPairMatrix(b *testing.B) {
+	trimmed := noWhitespace.ReplaceAllString(sampleHistogramPairMatrixPlain, "")
+	for i := 0; i < b.N; i++ {
+		s, err := json.Marshal(sampleHistogramPairMatrixValue)
+		if err != nil {
+			b.Fatal("error marshalling")
+		}
+
+		if string(s) != trimmed {
+			b.Fatalf("encoding error: expected %s, got %s", sampleHistogramPairMatrixPlain, s)
+		}
```

Comment:
We don't need this check in the benchmark; the correctness check belongs to unit tests, benchmark should only check performance. Can you remove this and re-run the benchmark and see if the numbers change? (either way, we should remove this)